docs/userguide/entry_point.rst

```diff
@@ -462,6 +472,14 @@ is useful:
        # do something with display
        ...
 
+Another point is that in this particular example, we have used plugins to
+customize the behaviour of a function (``display()``). In general, we can use entry
+points to enable plugins to not only customize the behaviour of functions, but also
+of entire classes and modules. This is unlike the case of console/GUI scripts,
+where entry points can only refer to functions. The syntax used for specifying the
+entry points remains the same as for console/GUI scripts, and is discussed in the
+`last section <#entry-points-syntax>`_.
+
 importlib.metadata
```

Comment:
Instead of having a section arbitrarily named after a stdlib package, how about we transform this part of the document in a `.. tip::` or `.. note::`?

Reply:
Ah, I see. But I think keeping it as a Tip or a Note might make users not read this part carefully - I mean, when there is some section of the document which is kept in a box and the font is smaller, people tend to skim over such things :)

How about changing the name of that section to something else like "Loading Entry Points"? What do you think about this?

Reply:
Or I guess another option might be to let go of that section altogether, and rephrase it like this perhaps:

> Finally, note that the`importlib.metadata` module is the recommended approach for loading and importing entry points. It is a part of the standard library since Python 3.8. For older versions of Python ...

so that it seems in continuation with the text above it.

In that case I guess we can also let go of that separate Summary section.

Reply:
I agree with your comments.

Actually the first thought I have was to rename the section to something like "How to load Entry Points?", which is inline of your proposal.

Then I realized that the previous section already introduced `importlib.metadata` in its examples and explicitly mention that it can be used to load the entry points...

My conclusion is that this part of the text is actually just a tip for users that `importlib-metadata` exists as a backport. It can likely be removed in the future (when 3.7 reaches EOL in June next year).

Please feel free to keep it as an independent section (but then we have to adequate the title to something like "Loading Entry Points in older versions of Python") or transform it in a `.. tip::/note::/important::` etc.

Reply:
I didn't look at it this way before - that the portion about the backport is going to gradually become less relevant for users, especially given that Python 3.7 will reach EOL in the near future. In that case I think you are right and it is better to convert that section into a Tip/Note/etc. I will also remove the Summary section header since a singular subsection under the section "Entry Points for Plugins" doesn't make sense.

Reply:
@abravalheri Have just made some changes. Let me know if you are happy with this.